scripts/slider.js

```diff
@@ -0,0 +1,28 @@
+let slideIndex = 1;
```

Comment:
Move this magic number at the top of the file and define it as constant

Reply:
But if I change to constant, then the slider doesn't work. Also this number is at the top of the file